runtimes/mlflow/mlserver_mlflow/runtime.py

```diff
@@ -196,5 +196,8 @@ def _sync_metadata(self) -> None:
 
     async def predict(self, payload: InferenceRequest) -> InferenceResponse:
         decoded_payload = self.decode_request(payload)
-        model_output = self._model.predict(decoded_payload)
+        params = None
+        if payload.parameters and payload.parameters.model_extra:
+            params = payload.parameters.model_extra
+        model_output = self._model.predict(decoded_payload, params=params)
```

Comment:
is there a test that can be added for this?

Reply:
@idlefella I think `params` is only supported for class-based pyfunc models and therefore this proposed changed will not work for function-based models.
I think we need to find a way to check the type of the model loaded and not pass `params` for function-based models.
check [here](https://mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#function-based-model-vs-class-based-model)

Reply:
Hi @sakoush 
Thanks for the comment. Ok then I will try to write a test for `class-based` and `function-based` models and see how or if these parameters can be forwarded. I hope to find time in the next couple of days.
Thanks!

Reply:
Hi @sakoush, when the underlying model is a python function, then the function is wrapped into a `_FunctionPythonModel`:
https://github.com/mlflow/mlflow/blob/7d5dcf7e9b59c0ed8799024cae9c054634c616cd/mlflow/pyfunc/model.py#L142
When params are set, the class prints a warning. So this doesn't cause any issues.
I also added some unittests for the forwarded params.